acceptance/bundle/artifacts/whl_change_version/script

```diff
@@ -0,0 +1,30 @@
+trace $CLI bundle deploy
+
+print_status() {
+    title "Expecting 1 wheel in libraries section in /jobs/create"
+    trace jq -s '.[] | select(.path=="/api/2.2/jobs/create") | .body.tasks' out.requests.txt
+
+    title "Expecting 1 wheel to be uploaded"
+    trace jq .path < out.requests.txt | grep import | grep whl | sort
+
+    rm out.requests.txt
+}
+
+trace find.py --expect 1 whl
+print_status
+
+title "Update glob to target 0.1.0 version\n"
+update_file.py databricks.yml './dist/*.whl' './dist/my*0.1.0*.whl'
```

Comment:
With this new glob it doesn't exercise the latest-version-matching code.

Reply:
Yes, this tests that old wheel is still present, as in https://github.com/databricks/cli/issues/2969

Still, good point, I changed the test to exercise new code as well.